### DISCUSSION TITLE
add `localized` to graphql

### DISCUSSION THREAD

**Discussion by soerenmeier** [craftcms/commerce#3783] [Ideas] [answered]
When building a language navigation it is important to know in which other languages the entry is available. To do this we use
```
query ($uri: [String], $siteId: [QueryArgument]) {
  entry(uri: $uri, siteId: $siteId) {
    localized {
      language
      url
    }
}
```

It would be useful to have the same `localized` property for products.
A workaround is to to do two separate queries:
```
  product(uri: $uri, siteId: $siteId) {
    ... productFields
  }
  productLocalized: product(uri: $uri, siteId: '*') {
    language
    url
  }
  ```
  
  Edit:
  On second though, i don't think this workaround works. I think a second query is needed after the id was received. Which increases pageload times.

**Comment by soerenmeier** [comment]
@lukeholder sorry to ping, but this would be really useful for us.

**Comment by nfourtythree** [comment, marked as answer]
Hi @soerenmeier 

Thank you for your suggestion! We can see how this would be useful when building out a project and we will look to get it in a future release of Commerce.

Thanks!

**Reply by lukeholder** [reply]
Craft Commerce 5.4 is now out with this fix/improvement. Thanks!

https://plugins.craftcms.com/commerce/changelog?craft5
